features/step_definitions/search_bar_steps.rb

```diff
@@ -0,0 +1,49 @@
+  Given('We have the following assignments exist:') do |table|
```

Comment:
This is duplicated from another step definition. Please remove.

Reply:
This does the same thing as `"the following assignments exist:"`

Reply:
Right, we can reuse the existing step definiton